synapse/storage/schema/main/delta/61/01insertion_event_lookups.sql

```diff
@@ -37,6 +37,15 @@ CREATE UNIQUE INDEX IF NOT EXISTS insertion_event_edges_event_id ON insertion_ev
 CREATE INDEX IF NOT EXISTS insertion_event_edges_insertion_room_id ON insertion_event_edges(room_id);
 CREATE INDEX IF NOT EXISTS insertion_event_edges_insertion_prev_event_id ON insertion_event_edges(insertion_prev_event_id);
 
+-- Add a table that keeps track of which "insertion" events need to be backfilled
+CREATE TABLE IF NOT EXISTS insertion_event_extremities(
+    event_id TEXT NOT NULL,
+    room_id TEXT NOT NULL
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS insertion_event_extremities_event_id ON insertion_event_extremities(event_id);
+CREATE INDEX IF NOT EXISTS insertion_event_extremities_room_id ON insertion_event_extremities(room_id);
+
```

Comment:
Bah, I don't know how I missed this, but this needs to be done in a new delta file, as we don't rerun delta files that have been run previously. (The only reason we tend to use `IF NOT EXISTS` is that it makes it easy to revert and rerun the delta when testing).

Reply:
Moved to `62` 👍

Reply:
Moving to `62` absolutely works, but just to note that you could have also moved it to a different file in `61/` and that would have also worked fine. (It's a bit handwavey when we should cut a new schema version, but 61 is getting a bit big so may as well do one now)

Reply:
Ahhh, I assumed we already shipped a Synapse version with `61` so people who already ran the upgrades wouldn't run deltas in that directory again. (probably what I just said is even wrong on how it works) ([docs](https://github.com/matrix-org/synapse/blob/develop/docs/development/database_schema.md))